libvast/include/vast/defaults.hpp

```diff
@@ -206,6 +206,9 @@ constexpr size_t disk_monitor_step_size = 1;
 /// Maximum number of events per INDEX partition.
 constexpr size_t max_partition_size = 1'048'576; // 1_Mi
 
+/// Timeout after which a partition is forcibly flushed.
+constexpr std::string_view partition_timeout = "1 hour";
```

Comment:
It cannot be `vast.import.partition-timeout` with the way our options are set up currently, because `vast import --partition-timeout` doesn't quite make sense.

I'm fine with `vast --partition-builder-timeout=... start`. Ultimately, I think @tobim should be the decider here. :-)

Reply:
I agree that specifying this at the client side doesn't make sense, because it's a server-side setting.

I would come from the perspective of "how should it look in the `vast.yaml` config of the server?"

Reply:
My first reaction was to keep it simple and go with `partition-timeout`. But then I checked the documentation and found that we do indeed explain the concept of an *active partition* in the architecture section, so I would agree to `active-partition-timeout` as well.

Reply:
Do we need to update that documentation to mention multiple active partitions?

Reply:
I implemented the renaming to `active-partition-timeout`.